web/assets/js/views/FeatureNotification.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function () {
+    $('#feature_notification_start_ts, #feature_notification_end_ts').pmiDateTimePicker();
+    $('#feature_notification_url').dropdownOther({
+        'All': '/*',
+        'Home Page': '/',
+        'Work Queue': '/workqueue/',
+        'Biobank Order Pages': '/participant/*/order/*',
+        'Physical Measurements': '/participant/*/measurements*',
+        'Management Tools': '/access/manage/dashboard',
+        'User Management': '/access/manage/user/groups'
```

Comment:
These need to be updated. Does Gwen have a list of pages she wants as dropdown options?

Reply:
Definitely, we need to remove the ones which has asterisk. Also, commented in the ticket asking Gwen to get the list.